src/styles/table.scss

```diff
@@ -240,6 +276,26 @@ $block: #{$fd-namespace}-table;
       @include fd-table-reset-spacing();
     }
 
+    &__group {
+      min-width: 100%;
+      border-left: none;
+
+      /**
+        Due to the changed table-layout group row cell may looe it's top border.
+        For example when previous (relatively to the group row) row columns tolal width < 100% of the table.
+      */
+      &::before {
+        content: '';
+        display: block;
+        position: absolute;
+        top: -0.0625rem;
+        left: 0;
+        right: 0;
```

Comment:
Shouldn't you handle RTL?

Reply:
what to handle here? `left` and `right` both are `0`